src/posts/lucky_1.0.0-rc1.cr

```diff
@@ -0,0 +1,239 @@
+class Lucky100rc1Release < BasePost
+  title "Lucky reaches the 1.0 ascent with the first release candidate"
+  slug "lucky-1_0_0-rc1-release"
+
+  def published_at : Time
+    Time.utc(year: 2022, month: 10, day: 1)
+  end
+
+  def summary : String
+    <<-TEXT
+    Lucky 1.0.0-rc1 has been released, and it's a huge milestone
+    for the framework and community.
+    TEXT
+  end
+
+  def content : String
+    <<-MD
+    As many developers know, reaching a 1.0 milestone is huge, and something to
+    be celebrated. We're stoked to release the first prelease of the 1.0 series with
+    `rc1`. We will be using this and the next few release candidates to clean up
```

Comment:
```suggestion
    As many developers know, reaching a 1.0 milestone is huge, and something to
    be celebrated. We're excited to publish the first release candidate of Lucky 1.0. 
    We will be using this and the next few release candidates to clean up
```

Just thinking "stoked" might not be a word most non-English speakers know and may not be easily looked up. So maybe "excited"? Up to you though!